docs/src/app/components/pages/components/Slider/ExampleValue.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Slider from 'material-ui/Slider';
+
+export default class SliderExampleValue extends React.Component {
+
+  state = {
+    firstSlider: 0.5,
+    secondSlider: 50,
+  }
+
+  handleSlider(key, value) {
+    this.setState({[key]: value});
+  }
+
+  render() {
+    return (
+      <div>
+        <Slider
+          defaultValue={0.5}
+          value={this.state.firstSlider}
+          onChange={(event, value) => this.handleSlider('firstSlider', value)}
```

Comment:
Well, I made this pull request because it was unclear how to get the value from the slider. 

So far as I can tell, _there's no way to actually get the value without putting a function in the render method._  This may be a problem that has to be resolved in the source of Slider.js, but that's not applicable here, this is just documenting *how* to get a value out of a slider. 

If there's a better way to do this, please let me know, because this method is the one I came up with after a half-day of beating myself over the head.  

If there's a better way to get the value, of course, that should replace this pull request.  

Here's the code from material-ui/src/Slider/Slider.js: 

```
//line 366
      // We need to use toFixed() because of float point errors.
      // For example, 0.01 + 0.06 = 0.06999999999999999
      if (this.state.value !== newValue) {
        this.setState({
          percent: newPercent,
          value: parseFloat(newValue.toFixed(5)),
        }, () => {
          if (this.props.onChange) this.props.onChange(event, this.state.value);
        });
      }

```
As far as I can tell, the only way to pass this.state.value to the handleChange method is via some sort of function. 

Perhaps a change like this may be needed to Slider.js: 

```
      if (this.state.value !== newValue) {
        this.setState({
          percent: newPercent,
          value: parseFloat(newValue.toFixed(5)),
        }, () => {
          if (this.props.onChange) this.props.onChange(Object.assign({}, event, {value: this.state.value}, value);
        });
      }
```
so as to not break backwards compatibility, but allow you to just simply bind the event, like so: 

```  handleSlider1(event) {
    this.setState({'slider1': event.value});
  }
// ...
          onChange={this.handleSlider1.bind(event)}
```

So, basically, what I'm saying is, so far as I can tell, there's actually no way to get the value from the slider without putting a function in the render method. :( If I'm wrong, I'd be pleasantly surprised!  :D